test/0.8.9/withdrawal-queue.test.js

```diff
@@ -7,10 +7,12 @@ const { MAX_UINT256, ACCOUNTS_AND_KEYS } = require('../helpers/constants')
 const { signPermit, makeDomainSeparator } = require('../0.6.12/helpers/permit_helpers')
 const { impersonate, EvmSnapshot, getCurrentBlockTimestamp, setBalance } = require('../helpers/blockchain')
 
+const ERC721ReceiverMock = artifacts.require('ERC721ReceiverMock')
+
 const { deployWithdrawalQueue } = require('./withdrawal-queue-deploy.test')
 
 contract('WithdrawalQueue', ([owner, stranger, daoAgent, user, pauser, resumer, oracle]) => {
-  let withdrawalQueue, steth, wsteth
+  let withdrawalQueue, steth, wsteth, normalizedShareRate
```

Comment:
Sorry, but what does it mean `normalized` in this context? Maybe `default` is better or make a structure with default args like @TheDZhon  suggested?